tests/kiwi_images_test/validate_build.pm

```diff
@@ -16,63 +24,46 @@ use warnings;
 use testapi;
 use utils;
 use version_utils "is_sle";
-my $sle_version = get_var('VERSION');
-my @files_list  = ('build.log');
+my $logfile = 'build.log';
 
-# check kiwi build status
-sub check_status {
-    my @builds = ("test-image-docker", "test-image-iso", "test-image-oem", "test-image-pxe", "test-image-vmx");
-    script_run("curl -k https://build.suse.de/project/monitor/QA:Maintenance:Images:$sle_version:kiwi-ng-testing > kiwi_out.html");
-    # sle12sp3 has 2 different builds
-    if (is_sle('<=12-SP3')) {
-        script_run("curl -k https://build.suse.de/project/monitor/QA:Maintenance:Images:$sle_version:kiwi-testing > kiwi_out2.html");
-        push @files_list, 'build2.log';
-    }
-    # parse build.log
-    foreach my $kiwi_build (@builds) {
-        script_run("(grep $kiwi_build kiwi_out.html | grep -q succeeded && echo \"SLE-$sle_version $kiwi_build PASSED\" || 
-        echo \"SLE-$sle_version $kiwi_build FAILED\") >> build.log");
-    }
+sub run {
+    my $sle_version  = get_var('VERSION');
+    my $install_type = get_var('KIWI_OLD');
 
-    # check run if old kiwi
-    if (is_sle('<=12-SP3')) {
-        script_run("(grep test-image-oem kiwi_out2.html | grep -q succeeded && echo \"SLE-$sle_version test-image-oem PASSED\" || 
-        echo \"SLE-$sle_version test-image-oem FAILED\") >> build2.log");
-        # show contents before check
-        script_run("cat build2.log");
+    # login in to system
+    assert_screen('kiwi_login', 120);
```

Comment:
Just noticed when you mentioned needle, why are you use this specific tag and new needle ?
tag `linux-login` should match

Reply:
I was not aware of this policy. Just tested and pushed a new commit using "linux-login" instead.

Reply:
no problem, but it is still better to use some generic needles than specific
one needle used in all tests instead x needles checking same screen